### DISCUSSION TITLE
I posted a question in General

### DISCUSSION THREAD

**Discussion by OldSubSailor** [Homebrew/discussions#396] [Getting started]
I don't want to duplicate, but I do think that it is a valid concern. I have gotten no responses, not even a 'you really need to post it ...'  The developers of said package/software said that it is a homebrew issue even, so I am at a loss at what to do.
I really would appreciate some help from you experts. 
Thank you.

**Comment by SMillerDev** [comment]
You don't need to keep making discussions, I guess nobody knows the answer then.
